reedsolomon/berlekampwelch_test.go

```diff
@@ -0,0 +1,159 @@
+package reedsolomon
+
+import (
+	"testing"
+
+	"github.com/HACKERALERT/infectious"
+	"github.com/stretchr/testify/require"
+	"go.dedis.ch/kyber/v4"
+	"go.dedis.ch/kyber/v4/group/edwards25519"
+)
+
+func checkEncoding(t *testing.T, encoded []Encoding, expected []byte, decoder RSDecoder) {
```

Comment:
as you've seen elsewhere, the `require` (and other libraries) function with a logic of 
`assertion(expected, actual, other parameters)`

Here you might want to do something like
```suggestion
func checkEncoding(t *testing.T, expected []byte, encoded []Encoding, decoder RSDecoder) {
```